packages/es-components/src/components/patterns/incrementer/Incrementer.js

```diff
@@ -79,15 +146,19 @@ function Incrementer(props) {
       >
         <ScreenReaderButtonText>
           Decrement value by
-          {props.decrementAmount}
+          {decrementAmount}
         </ScreenReaderButtonText>
         <Icon name="minus" />
       </RenderedButton>
       <IncrementerTextbox
         {...theme.validationInputColor.default}
-        type="text"
+        type="number"
```

Comment:
Should we add an `aria-label` to somehow let screen readers know what this incrementer represents? There is no associated label control right now, which I think was a misstep when I first wrote this component.

Reply:
Yep, I also didn't think to add that.

Reply:
Added `role="spinbutton"` and Home/End shortcut keys as per [WAI-ARIA practices](https://www.w3.org/TR/wai-aria-practices/#spinbutton). I also updated an example to show it being used with `Control` and `Label`.